src/pages/ReportSettingsPage.js

```diff
@@ -65,6 +66,31 @@ class ReportSettingsPage extends Component {
         ];
     }
 
+    /**
+     * @param {Object|null} linkedWorkspace - the workspace the report is on, null if the user isn't a member of the workspace
+     * @returns {Boolean}
+     */
+    shouldDisableRename(linkedWorkspace) {
```

Comment:
Can we just change this to currentWorkspace? I am not sure what linked workspace implies